tools/manage.sh

```diff
@@ -175,6 +176,12 @@ run_codespell () {
   "$POETRY_HOME/bin/poetry" run codespell
 }
 
+run_command () {
+  echo -e "${BIGreen}>>>${RST} Running ..."
+  shift;  # will remove first arg ("run") from the "$@"
+  "$POETRY_HOME/bin/poetry" run "$@"
```

Comment:
So, we're replacing `run` with `run`?

Reply:
Technically yes, I I thought it made more sense to keep it explicit.
* first `run` is purely arbitrary from `manage.sh` sub command choices and can be changed in the future
* second `run` here is dictated by `poetry`